### ISSUE TITLE
nfnets: question about use of Batch Normalization

### ISSUE THREAD

**Issue by purvang3** [deepmind/deepmind-research#174]
First of all, thank you for great publish nfnets. I have started deeging deep in to implementation, where I have some questions.

Unfortunately I am not able to run experiment.py. I am getting following error. I am running on just one gpu for testing.
<img width="1539" alt="Screen Shot 2021-02-18 at 6 40 53 PM" src="https://user-images.githubusercontent.com/39809304/108449785-ee6f8680-7218-11eb-9e28-a4a5f2540074.png">

 when I run test.py using fake data, it is working without any error. 

Also in ResBlockV2 class, code is still using batch normalization, and as per paper what I understood, it shouldn't.  
also couldn't find gradient clipping implementation in experiment.py. 
please correct me if I am misunderstanding? 
Thank you

**Comment by nss-ysasaki** [comment]
With a bit of digging I managed to get past the error above.

1. Add the following line to `experiment.py`
```
if __name__ == '__main__':
  FLAGS(sys.argv) # <- add this line
  flags.mark_flag_as_required('config')
  platform.main(Experiment, sys.argv[1:])
```
2. Add the following lines in the definition of `get_config()`, in `experiment.py`:
```
  config.save_checkpoint_interval = 60
  config.eval_specific_checkpoint_dir = ''
  config.checkpoint_dir = '/path/' # <- add this (modify /path/ appropriately)
  config.train_checkpoint_all_hosts = True # <- and this

  return config
```
3. Run `experiment.py` with `--config` argument, as follows:
```
python nfnets/experiment.py --config nfnets/experiment.py
```

The published version of deepmind/jaxline is outdated, perhaps?

PS: Even with this workaround, training halts with `TypeError`, but that's yet another issue...
